addons/wowl/static/dist/app_tests.js

```diff
@@ -1280,6 +1280,25 @@
             assert.ok(true);
         }
     });
+    QUnit$1.test("throw an error when there is a cycle in service dependencies", async (assert) => {
+        registry.add("a", {
+            name: "a",
+            dependencies: ['b'],
+            deploy: () => { },
+        });
+        registry.add("b", {
+            name: "b",
+            dependencies: ['a'],
+            deploy: () => { },
+        });
+        try {
+            await deployServices(env$2, registry);
+        }
+        catch (e) {
+            debugger;
+            assert.ok(true);
```

Comment:
need `assert.expect(1)` on top of file

Reply:
maybe assert with the beginning of the message. something like
`assert.ok(e.message.startsWith('...'));